packages/protocol/contracts/common/interfaces/IMintGoldScheduleInitializer.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IMintGoldScheduleInitializer {
+  function initialize(
+    uint256 _l2StartTime,
+    uint256 _communityRewardFraction,
+    address _carbonOffsettingPartner,
+    uint256 _carbonOffsettingFraction,
+    address registryAddress
```

Comment:
Yes, however it has been common practice to include the registry address as an initialze variable rather that hardcode it.

removed, since in this case `initialize()` no longer takes args.